docker/config/local_dev.env

```diff
@@ -32,3 +32,9 @@ CRASHPUBLISH_SUBSCRIPTION_NAME=local_dev_socorro_sub
 
 # Set Pub/Sub library to use emulator
 PUBSUB_EMULATOR_HOST=pubsub:5010
+
+# SQS things for local development
+CRASHPUBLISH_REGION=us-east-1
+CRASHPUBLISH_ACCESS_KEY=foo
+CRASHPUBLISH_SECRET_ACCESS_KEY=foo
+CRASHPUBLISH_ENDPOINT_URL=http://localstack-sqs:4576
```

Comment:
I tossed these in for now. They don't intersect with the Pub/Sub crashpublish settings.